packages/components/BrandText/BrandTextBase.tsx

```diff
@@ -26,12 +30,15 @@ export const BrandTextBase: React.FC<TextProps & { isTicker?: boolean }> = (
     );
 };
 
-// FIXME: remove StyleSheet.create
-// eslint-disable-next-line no-restricted-syntax
-const styles = StyleSheet.create({
-  base: {
-    color: "white",
-    fontSize: 20,
-    fontWeight: "600",
-  },
-});
+const useStyles = () => {
+  const isLightTheme = useSelector(selectIsLightTheme);
+  // eslint-disable-next-line no-restricted-syntax
+  const styles = StyleSheet.create({
```

Comment:
remove the stylesheet.create pls

Reply:
Why ?
There is a lot of `StyleSheet.create` all over the codebase

Reply:
(I changed it but i would have a reason)

Reply:
if you hover on the error it will explain the rationale

Reply:
main thing is that it breaks type safety

Reply:
one dev forced it into the repo then cargo cult made it appear everywhere, but I was against it since the start, we added the linter check recently because we decided to stop doing that

Reply:
Okay okay 👍 for type reason so